server/v2/cometbft/abci.go

```diff
@@ -226,11 +227,19 @@ func (c *Consensus[T]) InitChain(ctx context.Context, req *abciproto.InitChainRe
 		})
 	}
 
+	ci, err := c.store.LastCommitID()
+	if err != nil {
+		return nil, err
+	}
+
+	// populate hash with empty byte slice instead of nil
+	bz := sha256.Sum256([]byte(""))
```

Comment:
Is `[]byte("")==[]byte{}`? https://github.com/cosmos/cosmos-sdk/blob/main/store/rootmulti/store.go#L450

I guess yes, but maybe we want to keep it consistent

Reply:
modified to be consistent, thank you